integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/apim/utils/PublisherUtils.java

```diff
@@ -163,16 +133,18 @@ public static String createAPI(APIRequest apiRequest, RestAPIPublisherImpl publi
         try {
             createAPIResponse = publisherRestClient.addAPI(apiRequest);
         } catch (ApiException e) {
+            log.error(e.getMessage());
```

Comment:
Shall we include the error message as error while adding the API, etc?

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2624/commits/8f3da3bc19b02e60e2946ddaf269f381f33e527d